pyuvsim/antenna.py

```diff
@@ -64,7 +66,7 @@ def get_beam_jones(self, array, source_alt_az, frequency, reuse_spline=True,
                                                  reuse_spline=reuse_spline)
         Ncomponents = source_za.shape[-1]
 
-        # interp_data has shape: (Naxes_vec, Nspws, Nfeeds, 1 (freq),  Ncomponents (source positions))
+        # interp_data has shape: (Naxes_vec, Nspws, Nfeeds, 1 (freq),  Ncomponents (source pos))
```

Comment:
This feels to me like a case in which a shorter line hurts readability.

Reply:
sure, I'll fix that.